### DISCUSSION TITLE
kpanda/user-guide/clusters/upgrade-cluster/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1289] [Announcements]
# kpanda/user-guide/clusters/upgrade-cluster/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/user-guide/clusters/upgrade-cluster/

**Comment by liyuerich** [comment]
集群升级选择的是 lengrongfu，确认的时候是 skoala-dev， 看实时日志是michelle-01集群。比较乱，能统一一下吗？
